### DISCUSSION TITLE
[community] bacalhau version question

### DISCUSSION THREAD

**Discussion by likhita-8091** [Lilypad-Tech/lilypad#287] [Q&A]
why display error? 
![image](https://github.com/user-attachments/assets/7ecf276c-7903-4935-92d3-0e6fa73a75ed)

**Comment by developersteve** [comment]
@likhita-8091 Thank you for posting. It looks like you are using the Windows deployment; I just wanted to confirm which version of Bacalhau you are using?

**Comment by likhita-8091** [comment]
> @likhita-8091 Thank you for posting. It looks like you are using the Windows deployment; I just wanted to confirm which version of Bacalhau you are using?

yes,i clone source code, run the windows env。the bacalhau version is v1.3.2

**Comment by AquiGorka** [comment]
@likhita-8091 are you sure that's `bacalhau@1.3.2`? I've noticed that error when trying to run `1.4.0`, can you post the output from `bacalhau version`?
